.changeset/hungry-oranges-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontity/amp": patch
```

Comment:
Yeah, I think this is fine. Either a `patch` or a `minor`, it's not very important.

@santosguillamot, we are going to release the `@frontity/amp` package in "beta" first, right? So `0.0.1` is fine I mean.

Reply:
Yes. The idea was to release a beta with the first functionalities and start testing it.